client/app/views/dashboard/screenshots/screenshots.html

```diff
@@ -91,7 +91,9 @@
       max-size="maxSize"
       rotate="false"
       boundary-links="true"
-      items-per-page="itemsPerPage">
+      items-per-page="itemsPerPage"
+      ng-show="screenshots.length > itemsPerPage">
```

Comment:
The correct way should be to check if in the screenshots  RESTful results we have a `next` key